src/index.scss

```diff
@@ -19,7 +19,6 @@ $fa-font-path: "~font-awesome/fonts";
 @import "./components/RequestCodesPage/RequestCodesPage";
 @import "./components/Sidebar/Sidebar";
 @import "./components/subscriptions/styles/";
-@import "./components/StatusAlert/StatusAlert";
 @import "./components/LoadingMessage/LoadingMessage";
 @import "./components/TableComponent/TableComponent";
```

Comment:
Could/should we also remove the custom `TableComponent` and `TableLoadingOverlay` styles here as well?